db/seed/beneficiary.seed.ts

```diff
@@ -0,0 +1,45 @@
+import { PrismaClient, PersonRelation } from '@prisma/client'
+const prisma = new PrismaClient()
+
+export async function beneficiarySeed() {
+  console.log('Beneficiary seed')
+
+  const persons = await prisma.person.findMany({ where: { newsletter: true } })
```

Comment:
What's the correlation between people that are subscribed to newsletter and beneficiary seed?

Reply:
that's true I'll get them all